tests/graph/test_op.py

```diff
@@ -203,7 +212,7 @@ def test_get_test_values_success():
 def test_get_test_values_exc():
     """Tests that `get_test_values` raises an exception when debugger is set to raise and a value is missing."""
 
-    with pytest.raises(TestValueError):
+    with pytest.raises(TestValueError) and pytest.warns(FutureWarning):
```

Comment:
Can you `and` contexts like this? I always nest them, but if it works fine. Did you test it/ see it working like this anywhere?

Reply:
No I think this might be wrong. The tests passed so I assumed it works but turns out ```with``` statements must be nested.